src/lib/configs/project/service.ts

```diff
@@ -0,0 +1,161 @@
+  },
+  required: ["get"],
+};
+
+export const FACADE_MODULE_NAME = "facade";
+
+export type ConfigV0 = {
+  version: 0;
+  modules: { [FACADE_MODULE_NAME]: ModuleV0 } & Record<string, ModuleV0>;
+};
+
+const configSchemaV0: JSONSchemaType<ConfigV0> = {
+  type: "object",
+  properties: {
+    version: { type: "number", enum: [0] },
+    modules: {
+      type: "object",
+      additionalProperties: moduleSchema,
+      properties: {
+        [FACADE_MODULE_NAME]: moduleSchema,
+      },
+      required: [FACADE_MODULE_NAME],
+    },
+  },
+  required: ["version", "modules"],
+};
+
+const migrations: Migrations<Config> = [];
+
+const examples = `
+modules:
+  facade:
+    get: modules/facade
+
+    # Overrides for module:
+    maxHeapSize: "100" # 100 bytes
+    # maxHeapSize: 100K # 100 kilobytes
+    # maxHeapSize: 100 Ki # 100 kibibytes
+    # Max size of the heap that a module can allocate in format: <number><whitespace?><specificator?>
+    # where ? is an optional field and specificator is one from the following (case-insensitive):
+    # K, Kb - kilobyte; Ki, KiB - kibibyte; M, Mb - megabyte; Mi, MiB - mebibyte; G, Gb - gigabyte; Gi, GiB - gibibyte;
+    # Current limit is 4 GiB
+    loggerEnabled: true # true, if it allows module to use the Marine SDK logger.
+    loggingMask: 0 # manages the logging targets, described in here: https://doc.fluence.dev/marine-book/marine-rust-sdk/developing/logging#using-target-map
+    mountedBinaries:
+      curl: /usr/bin/curl # a map of mounted binary executable files
+    preopenedFiles: # a list of files and directories that this module could access with WASI
+      - ./dir
+    volumes: # a map of accessible files and their aliases.
+    # Aliases should be normally used in Marine module development because it's hard to know the full path to a file.
+      aliasForSomePath: ./some/path
+    envs: # environment variables accessible by a particular module with standard Rust env API like this std::env::var(IPFS_ADDR_ENV_NAME).
+      # Please note that Marine adds three additional environment variables. Module environment variables could be examined with repl
+      ENV1: arg1
+      ENV2: arg2`;
+
+type Config = ConfigV0;
+type LatestConfig = ConfigV0;
+export type ServiceConfig = InitializedConfig<LatestConfig>;
+export type ServiceConfigReadonly = InitializedReadonlyConfig<LatestConfig>;
+
+const getInitConfigOptions = (
+  configDirPath: string
+): InitConfigOptions<Config, LatestConfig> => ({
+  allSchemas: [configSchemaV0],
+  latestSchema: configSchemaV0,
+  migrations,
+  name: SERVICE_CONFIG_FILE_NAME,
+  getSchemaDirPath: ensureFluenceDir,
+  getConfigDirPath: (): string => configDirPath,
+  examples,
+});
+
+export const initServiceConfig = (
+  configDirPath: string,
+  commandObj: CommandObj
+): Promise<InitializedConfig<LatestConfig> | null> =>
+  getConfigInitFunction(getInitConfigOptions(configDirPath))(commandObj);
+export const initReadonlyServiceConfig = (
```

Comment:
new line

Reply:
what's the problem?

Reply:
You want to add new line before export, so it looks nicer?